cg/store/filters/status_order_filters.py

```diff
@@ -18,15 +18,15 @@ def filter_orders_by_ids(orders: Query, ids: list[int] | None, **kwargs) -> Quer
     return orders.filter(Order.id.in_(ids)) if ids else orders
 
 
-def apply_limit(orders: Query, limit: int | None, **kwargs) -> Query:
-    return orders.limit(limit) if limit else orders
+def apply_pagination(orders: Query, page: int | None, page_size: int | None, **kwargs) -> Query:
+    return orders.limit(page_size).offset((page - 1) * page_size) if page and page_size else orders
```

Comment:
Consider breaking out into a separate method.

Reply:
A doc string with a little explanation on what happens here would be great.

Reply:
Not sure where to put it, I'll leave it for now. Agree that it can be made into a reusable general function if needed elsewhere